.github/workflows/workflow.yml

```
@@ -48,7 +48,7 @@ jobs:
 
       - name: Coverage monitor
         if: env.EXECUTE_COVERAGE == 'true'
-        uses: slavcodev/coverage-monitor-action@1.1.0
+        uses: slavcodev/coverage-monitor-action@1.8.0
```

```suggestion
        uses: slavcodev/coverage-monitor-action@v1
```

if you'd like the latest version

---

Thanks!